hpOneView/resources/settings/scopes.py

```diff
@@ -168,28 +130,5 @@ def update_resource_assignments(self, id_or_uri, resource_assignments, timeout=-
         Returns:
             dict: Updated resource.
         """
-        uri = self._client.build_uri(id_or_uri) + "/resource-assignments"
-
-        headers = {'Content-Type': 'application/json'}
-
-        return self._client.patch_request(uri, resource_assignments, timeout=timeout, custom_headers=headers)
-
-    def patch(self, id_or_uri, operation, path, value, timeout=-1):
-        """
-        Uses the PATCH to update a resource for the given scope.
-
-        Only one operation can be performed in each PATCH call.
-
-        Args:
-            id_or_uri: Can be either the resource ID or the resource URI.
-            operation: Patch operation
-            path: Path
-            value: Value
-            timeout: Timeout in seconds. Wait for task completion by default. The timeout does not abort the operation
-                in OneView; it just stops waiting for its completion.
-
-        Returns:
-            dict: Updated resource.
-        """
-        headers = {'Content-Type': 'application/json-patch+json'}
-        return self._client.patch(id_or_uri, operation, path, value, timeout=timeout, custom_headers=headers)
+        uri = self._helper.build_uri(id_or_uri) + "/resource-assignments"
+        return super(Scopes, self).patch_request(uri, resource_assignments, timeout=timeout)
```

Comment:
What is "resource_assignments" here..?
Please refer to resource.py file

Reply:
This URI supported only till API version 300, "resource-assignments" takes existing resource uri's and updates the scope.